.travis.yml

```diff
@@ -31,7 +36,11 @@ script:
   - npm test
   - python -m pytest --cov
   - codecov
-  - docker run --volume=$(pwd):/app --workdir=/app coala/base coala-ci
+  - docker run --volume =$(pwd): / app --workdir = /app coala/base coala-ci
```

Comment:
I think you shouldn't change this?
See https://travis-ci.org/coala/coala-html/builds/311194465#L686

Reply:
@nalinbhardwaj

Reply:
Just noticed the command is parsed like YAML